testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/EEConcurrencyExecutorShutdownTestCase.java

```diff
@@ -0,0 +1,230 @@
+    public void removeTestResources() throws IOException, MgmtOperationException {
+        try {
+            // /profile=default/subsystem=ee/context-service=default:remove()
+            ModelNode removeContextService = createOpNode("profile=default/subsystem=ee/context-service=default", REMOVE);
+            executeForResult(removeContextService, masterClient);
+            // /profile=default/subsystem=ee/managed-executor-service=default:remove()
+            ModelNode removeManagedExecutorService = createOpNode("profile=default/subsystem=ee/managed-executor-service=default", REMOVE);
+            executeForResult(removeManagedExecutorService, masterClient);
+        } finally {
+            testSupport.stop();
+        }
+    }
+
+    @AfterClass
+    public static void tearDownDomain() throws Exception {
+        try {
+            testSupport = null;
+            domainMasterLifecycleUtil = null;
+            DomainTestSuite.stopSupport();
+        } finally {
+            cleanFile(tmpDir);
+        }
+    }
+
+    private void addTestResources() throws IOException, MgmtOperationException {
+        // /profile=default/subsystem=ee/context-service=default:add(jndi-name=java:jboss/ee/concurrency/context/default, use-transaction-setup-provider=true)
+        ModelNode addContextService = createOpNode("profile=default/subsystem=ee/context-service=default", ADD);
+        addContextService.get("jndi-name").set("java:jboss/ee/concurrency/context/default");
+        addContextService.get("use-transaction-setup-provider").set(true);
+        executeForResult(addContextService, masterClient);
+        // /profile=default/subsystem=ee/managed-executor-service=default:add(jndi-name=java:jboss/ee/concurrency/executor/default, context-service=default, hung-task-threshold=60000, keepalive-time=5000)
+        ModelNode addManagedExecutorService = createOpNode("profile=default/subsystem=ee/managed-executor-service=default", ADD);
+        addManagedExecutorService.get("jndi-name").set("java:jboss/ee/concurrency/executor/default");
+        addManagedExecutorService.get("context-service").set("default");
+        addManagedExecutorService.get("hung-task-threshold").set(60000);
+        addManagedExecutorService.get("keepalive-time").set(5000);
+        executeForResult(addManagedExecutorService, masterClient);
+    }
+
+    /**
+     * Tests if the server with running ConcurrencyExecutor can be stopped using cli command
+     * /host=master/server-config=server-one:stop(timeout=0)
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testConcurrencyExecutorShutdown() throws Exception {
+        ModelNode content = new ModelNode();
+        content.get("archive").set(true);
+        content.get("path").set(new File(tmpDir, "archives/" + ARCHIVE_FILE_NAME).getAbsolutePath());
+        ModelNode deploymentOpMain = createDeploymentOperation(content, MAIN_SERVER_GROUP_DEPLOYMENT_ADDRESS);
+        executeOnMaster(deploymentOpMain);
```

Comment:
You could use `executeForResult(deploymentOpMain, masterClient);` instead of `executeOnMaster(deploymentOpMain);`. The former will check that the operation was correctly executed, the latter has no benefits. If you do so, you could even remove completely the `executeOnMaster` method from the test, currently, it is only used here.